phoenix-core/src/main/java/org/apache/phoenix/schema/types/PDouble.java

```diff
@@ -17,15 +17,16 @@
  */
 package org.apache.phoenix.schema.types;
 
-import com.google.common.base.Preconditions;
-import com.google.common.primitives.Doubles;
```

Comment:
Minor, but can you import into Eclipse our phoenix/dev/phoenix.importorder and re-save these files so that the import order doesn't change?